CHANGELOG.md

```diff
@@ -1,3 +1,97 @@
+### citus v10.1.0 (June 15, 2021) ###
+
+* Drops Postgres 11 support
+
+* Fixes a bug that can cause a crash when DEBUG4 logging is enabled
+
+* Fixes column default coming from a sequence
+
+* Fixes data race in `get_rebalance_progress`
+
+* Fixes error message for local table joins
+
+* Fixes FROM ONLY queries on partitioned tables
+
+* Fixes issues caused by public schema being omitted in queries
+
+* Fixes matview access method change issue
+
+* Fixes nested select query with union bug
+
+* Fixes possible segfaults when using Citus in the middle of an upgrade
+
+* Fixes problems with concurrent calls of `DropMarkedShards`
+
+* Fixes relname null bug when parallel execution
+
+* Fixes shared dependencies that are not resident in a database
+
+* Fixes stale hostnames bug in prep. statements after `master_update_node`
+
+* Fixes using 2PC when it might be necessary
+
+* Adds `shard_count` parameter to `create_distributed_table` function
+
+* Adds support for `ALTER DATABASE OWNER`
+
+* Adds support for temporary columnar tables
+
+* `alter_columnar_table_set` enforces columnar table option constraints
+
+* Checks free disk space before moving a shard
+
+* Closes connection after each shard move
```

Comment:
Also putting some suggestions:

Closes connection after each shard move ->
Makes sure that connection is closed after each shard move

Checks free disk space before moving a shard ->
Ensures free disk space before moving a shard

Does smarter cleanup of orphaned shards ->
Improves orphaned shard cleanup logic

Errors out if inheriting a distributed table ->
Prevents inheriting a distributed table 

Executor refrains from opening extra connections->
Avoids opening extra connections

Removes limits around long partition names ->
Removes length limits around partition names

Fixes issues caused by public schema being omitted in queries ->
Fixes issues caused by omitting public schema in queries

Fixes matview access method change issue ->
Preserves access method of materialized views when undistributing or altering distributed tables

Reduces memory usage while rebalancing ->
Reduces memory usage while rebalancing shards

Reply:
This part is a form of art